--- a/Cargo.toml	2024-04-23 11:18:39.986117584 -0600
+++ b/Cargo.toml	2024-04-23 11:18:21.704093187 -0600
@@ -114,9 +114,13 @@
 dinit = []
 
 [profile.release]
-debug = "line-tables-only"
+codegen-units = 1
+debug = 0
+opt-level = 3
 overflow-checks = true
-lto = "thin"
+lto = "fat"
+panic = "abort"
+strip = true
 
 [profile.release.package.niri-config]
 # knuffel with chomsky generates a metric ton of debuginfo.
